receiver/k8sclusterreceiver/documentation.md

```diff
@@ -138,15 +138,15 @@ Total number of available pods (ready for at least minReadySeconds) targeted by
 
 | Unit | Metric Type | Value Type |
 | ---- | ----------- | ---------- |
-| 1 | Gauge | Int |
+| {pods} | Gauge | Int |
```

Comment:
Please use the singular form. This is the latest recommendation in the OTel spec
```suggestion
| {pod} | Gauge | Int |
```